qlib/utils/__init__.py

```diff
@@ -280,6 +280,9 @@ def parse_field(field):
         (r"(\w+\s*)\(", r"Operators.\1("),
     ]:  # Features  # Operators
         field = re.sub(pattern, new, field)
+    #for ChangeInstrument
```

Comment:
I don't think this substitution.
If you don't add this lines of code. I think the expression below will work
"ChangeInstrument('SH600519',$close)"

Reply:
Without the two lines below, I would see the following error for the following code: 
marketRet = ChangeInstrument('SH000300', Feature('close')/Ref(Feature('close'),1) -1)
and use QlibDataLoader to load the values.

expression = eval(parse_field(field))
  File "<string>", line 1, in <module>
NameError: name 'SH000300' is not defined.

It's likely related to how quotes are dealt with in the sub function, but I didn't dive into it. With the two lines below, the error was gone. If you know the exact reason, please let me know. Thanks!

Reply:
I remove the two lines and implemented a test. It passes.
Could you give a examples or test about how to reproduce your error ?
Thanks